src/replication_test.c

```diff
@@ -52,6 +52,7 @@ __thread char  tinfo[50] =  {0};
 	mgmt_ack_data->pool_guid = replica_port;\
 	mgmt_ack_data->checkpointed_io_seq = 1000;\
 	mgmt_ack_data->zvol_guid = replica_port;\
+	strcpy(mgmt_ack_data->replica_id, replica_id);\
```

Comment:
memset to 0

Reply:
this is required as the entire code in istgt uses strncmp to compare all 32 bytes